projects/cashmere/src/lib/radio-button/radio-button.component.ts

```diff
@@ -117,10 +126,18 @@ export class RadioButtonComponent implements OnInit {
         }
     }
 
+    /**
+     * Function for binding to the click event
+     * @param event - event to be passed through to the handler
+     */
     onInputClick(event: Event) {
```

Comment:
I think `onInputClick()` and `onInputChange()` would be candidates to be prefixed with an underscore. There used by the template, but a consumer shouldn't need them